Master Project/Assets/Scenes/Combat/Qi_Scripts/MonsterAction.cs

```diff
@@ -2,14 +2,127 @@
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.UI;
+using System;
 
 public class MonsterAction : MonoBehaviour {
 
-    public enum Monster {Nessie, Cerberus, REDACTED};
-    public Animator Hitted;
+    System.Random monsterdice = new System.Random();
+    System.Random damagedice = new System.Random();
+    private int monsteraction;
+    private bool monstermoved;
+    private int damageholder;
 
-    public void MonsterHitted()
+    public enum Monster { Nessie, Cerberus, REDACTED };
+    public Monster monster;

```

Comment:
These class attributes do not have a consistent naming convention... Probably best to follow Unity standards which, for public attributes, is `camelCase`.